container/streams/streams.go

```diff
@@ -0,0 +1,37 @@
+package streams
```

Comment:
we don't need this package anymore. The streams where extracted to `runconfig` in #18051:

https://github.com/docker/docker/pull/18051/files#diff-1cef1c6341912d280785a2ed38bf3e2f